pickapp/src/main/kotlin/com/lovoo/android/pickapp/adapter/SelectionAdapter.kt

```diff
@@ -72,14 +71,17 @@ class SelectionAdapter : RecyclerView.Adapter<SelectionAdapter.ViewHolder>() {
      */
     fun get(index: Int) = list.getOrNull(index)
 
+    fun getListCount() = list.size
+
     /**
      * @param uri the [Uri] that should be added to the selection at list start
      * @return always true
      */
-    fun add(uri: Uri): Boolean {
-        list.add(0, uri)
-        notifyItemInserted(0)
-        return true
+    fun add(uri: Uri): Int {
+        val position = list.size
+        list.add(uri)
+        notifyItemChanged(position)
+        return position
```

Comment:
This means it is now added to list end and not start anymore, right?

Reply:
I guess so, the commit description says it removed the reversed order.